<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Akhilesh" failures="1" tests="4" name="com.github.test.SearchTest" time="6.350" errors="0" timestamp="2022-07-16T13:28:06 IST" skipped="0">
  <testcase classname="com.github.test.SearchTest" name="validate_most_starred_seleniium_java_repository" time="1.771"/>
  <system-out/>
  <testcase classname="com.github.test.SearchTest" name="validate_python_seleniium_repository_count" time="1.031"/>
  <system-out/>
  <testcase classname="com.github.test.SearchTest" name="validate_python_and_java_seleniium_repository_count" time="1.771"/>
  <system-out/>
  <testcase classname="com.github.test.SearchTest" name="validate_java_seleniium_repository_count" time="1.777">
    <failure message="Expected and Actual repo count is not matching for selenium with Java repo. expected [54009] but found [54035]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Expected and Actual repo count is not matching for selenium with Java repo. expected [54009] but found [54035]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at com.github.test.SearchTest.validate_java_seleniium_repository_count(SearchTest.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
    </failure>
  </testcase> <!-- validate_java_seleniium_repository_count -->
  <system-out/>
</testsuite> <!-- com.github.test.SearchTest -->
